.github/workflows/main.yml

```diff
@@ -30,6 +30,20 @@ jobs:
             - name: Install Node.js dependencies
               run: npm i
 
-            - name: Run linters
-              continue-on-error: false
+            - name: Pull starknet-hardhat-plugin
+              run: npm run link-plugin
+
+            - name: Run TypeScript correctness
+              run: npx tsc --noEmit
+
+            - name: Check for executables
+              run: npm run check-executable
+
+            - name: Check for "it.only" occurrence in files
+              run: npm run check-it-only-file
```

Comment:
Here we are just informing the reader of some implementation detail. The actual purpose of this step is to "check all tests are executed". We should rename this step and the script.

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-example/pull/122/commits/163aa1540a0b7037ca1ee2f6efbcff1f41e5a0e3